source/containers.tex

```diff
@@ -3164,6 +3164,12 @@
   template<class T, size_t N>
     constexpr void swap(array<T, N>& x, array<T, N>& y) noexcept(noexcept(x.swap(y)));
 
+  // \ref{array.creation}, Array creation functions
```

Comment:
I think this should be "array", un-capitalized.

Reply:
Un-cap'ed and \tcoded.